udp-native/src/main/java/org/ballerinalang/stdlib/udp/SelectorManager.java

```diff
@@ -267,16 +268,16 @@ private void processError(ReadPendingCallback callback, SocketConstants.ErrorTyp
     }
 
 
-    private BArray createUdpSocketReturnValue(ReadPendingCallback callback, byte[] bytes,
-            InetSocketAddress remoteAddress) {
+    private BMap<BString, Object>  createUdpSocketReturnValue(ReadPendingCallback callback, byte[] bytes,
+                                                             InetSocketAddress remoteAddress) {
         BMap<BString, Object> address = ValueCreator.createRecordValue(SOCKET_PACKAGE_ID, "Address");
         address.put(StringUtils.fromString("port"), remoteAddress.getPort());
         address.put(StringUtils.fromString("host"), StringUtils.fromString(remoteAddress.getHostName()));
         BArray contentTuple = ValueCreator.createTupleValue(receiveFromResultTuple);
-        contentTuple.add(0, ValueCreator.createArrayValue(bytes));
-        contentTuple.add(1, Long.valueOf(callback.getCurrentLength()));
-        contentTuple.add(2, address);
-        return contentTuple;
+        BMap<BString, Object> datagram = ValueCreator.createRecordValue(SOCKET_PACKAGE_ID, "Datagram");
```

Comment:
Move `Datagram` to a constant.

Reply:
"Datagram" replaced with SocketConstants.DATAGRAM